tests/unit/phpDocumentor/Compiler/Linker/LinkerTest.php

```diff
@@ -30,6 +30,8 @@
  * Tests the functionality for the Linker class.
  *
  * @coversDefaultClass \phpDocumentor\Compiler\Linker\Linker
+ * @covers ::__construct
+ * @covers ::<protected>
```

Comment:
protected? Should this not be private and/or the protected methods in the Linker set to private?